disperser/controller/signature_receiver.go

```diff
@@ -156,16 +156,15 @@ func (sr *signatureReceiver) receiveSigningMessages(ctx context.Context, attesta
 	operatorCount := len(sr.indexedOperatorState.IndexedOperators)
 
 	// we expect a single SigningMessage from each operator
+forLoop:
```

Comment:
This is really weird, but I guess it makes sense. Linter enforces this?

Reply:
yes. Why is it weird? I think its cleaner than the previous thing which is more bug-prone (have another break variable that you need to remember). Whereas here all you have is a labelled break statement (need to remember to add the label though).